feconf.py

```diff
@@ -1189,15 +1189,6 @@ def get_empty_ratings():
     }
 }
 
-# A dict representing available landing pages, having subject as a key and list
-# of topics as the value.
-# Note: This dict needs to be keep in sync with frontend TOPIC_LANDING_PAGE_DATA
-# oppia constant defined in
-# core/templates/pages/landing-pages/TopicLandingPage.js file.
-AVAILABLE_LANDING_PAGES = {
```

Comment:
Where did this move to?

Reply:
This is moved to constants.ts so, this can be shared with the frontend too.